### PR TITLE
[PBA-6293] Added logic for hide home indicator on iPhone X

### PR DESCRIPTION
**Pull request by MarkEPAM**
- PR resolves problem with showed home indicator on iPhone X. 
- Were added functions for view controllers. 
- PR without unit tests, because they are not required.

### CODE REVIEW

**Review by brol1dev — APPROVED** [review]
LGTM

File: sdk/iOS/OoyalaSkinSDK/ViewControllers/OOSkinFullScreenViewController.m [hunk 1]
```diff
@@ -38,6 +38,10 @@ - (instancetype)init {
 
 #pragma mark - Override view controller functions
 
+- (BOOL)prefersHomeIndicatorAutoHidden {
+  return YES;
```
Comment (pilievOoyala) [inline sdk/iOS/OoyalaSkinSDK/ViewControllers/OOSkinFullScreenViewController.m:42]: Do we need to do some kind of check to see if this is an iPhoneX? If not, will this effect older phones like the 8.  What about future phones?

File: sdk/iOS/OoyalaSkinSDK/ViewControllers/OOSkinViewController.m [hunk 2]
```diff
@@ -201,6 +201,10 @@ - (void)dealloc {
 
 #pragma mark - Override view controller functions
 
+- (BOOL)prefersHomeIndicatorAutoHidden {
+  return YES;
```
Comment (pilievOoyala) [inline sdk/iOS/OoyalaSkinSDK/ViewControllers/OOSkinViewController.m:205]: Same thing here.

File: sdk/iOS/OoyalaSkinSDK/ViewControllers/OOSkinFullScreenViewController.m (hunk 1 above)
Comment (IvanSakharovskii) [reply sdk/iOS/OoyalaSkinSDK/ViewControllers/OOSkinFullScreenViewController.m:42]: Only iPhone X have home indicator. For previous versions this function won't be called.

Comment (brol1dev) [reply sdk/iOS/OoyalaSkinSDK/ViewControllers/OOSkinFullScreenViewController.m:42]: it will only be called for iPhone X @pilievOoyala

**Comment by pilievOoyala** [comment]
Looks good. Thank you for clarifying.

**Pull request merged**